server/index.js

```diff
@@ -1,25 +1,49 @@
-var express = require('express');
-var bodyParser = require('body-parser');
-var db = require('../database');
-var path = require('path');
+const express = require('express');
+const bodyParser = require('body-parser');
+const model = require('./model/model.js');
+const path = require('path');
+const port = process.env.PORT || 7000;
 
 const app = express();
 app.use(express.static(__dirname + '/../react-client/dist'));
+app.use(bodyParser.json());
 
 app.all('/*', function(req, res, next) {
   res.header('Access-Control-Allow-Origin', '*');
   next();
 });
 
 app.get('/description', function(req, res) {
-  const id = Number(req.query.id);
-	model.getListing(id)
+	model.getListing(req.query.id)
 		.then((results) => {
 			res.send(results);
 		})
 		.catch((err) => console.log(err));
 });
 
+app.post('/description', function(req, res) {
+	model.addListing(req.body)
+		.then(() => {
+			res.send();
```

Comment:
Definitely great that you are catching any errors everywhere. 

I'm not sure whether it would be best practice or not to add in -- is the default `response` from the DB post operation worth sending?